### ISSUE TITLE
Appears to be incompatible with Redactor 10

### ISSUE THREAD

**Issue by JonFabritius** [tr42/redactor-mentions#15]
Lots of changes in Redactor 10; This plugin doesn't appear to work with the latest version.

**Comment by yourfavorite** [comment]
Nope. Its going to have to be reworked quite a bit due to the new structure of Redactor. No ETA on this either. Sorry.

**Comment by jasonkeene** [comment]
Humm, I just remembered I needed to work on this. Adding it to my todo list.

**Comment by thebitguru** [comment]
Hi Jason,

FYI, I have my fork of this project updated to support Redactor 10, but my changes are not backwards compatible.  My fork also addresses #14 and adds support for installation using bower.  I am hoping to have my updates finalized within the next few days and will submit a pull request.

I may able to help if you need any assistance maintaining this project.  Feel free to add me as a collaborator if you are comfortable.
